c/examples/api_structure.c

```diff
@@ -2,25 +2,24 @@
 #include <stdlib.h>
 #include <tskit/tables.h>
 
-#define check_tsk_error(val) if (val < 0) {\
-    fprintf(stderr, "line %d: %s", __LINE__, tsk_strerror(val));\
-    exit(EXIT_FAILURE);\
-}
+#define check_tsk_error(val)                                                   \
+  if (val < 0) {                                                               \
+    fprintf(stderr, "line %d: %s", __LINE__, tsk_strerror(val));               \
+    exit(EXIT_FAILURE);                                                        \
+  }
 
-int
-main(int argc, char **argv)
-{
-    int j, ret;
-    tsk_edge_table_t edges;
+int main(int argc, char **argv) {
+  int j, ret;
+  tsk_edge_table_t edges;
```

Comment:
IMO having the function name start on a newline makes the code more grep friendly: `grep -n ^main *.c`.

And changing the indent settings to use 4 spaces will probably reduce this diff dramatically.